src/unix/notbsd/linux/mod.rs

```diff
@@ -748,7 +748,7 @@ cfg_if! {
         impl ::fmt::Debug for af_alg_iv {
             fn fmt(&self, f: &mut ::fmt::Formatter) -> ::fmt::Result {
                 f.debug_struct("af_alg_iv")
-                    // .field("iv", self.as_slice())
+                    .field("iv", &self.as_slice().iter().collect::<Vec<_>>())
```

Comment:
I think that &self.as_slice() should be enough, no need to call iter and collect (which would allocate memory).

Reply:
Unfortunately, it doesn't compile with error `doesn't have a size known at compile-time`